src/Core/Contact/Infrastructure/Repository/DbReadContactTemplateRepository.php

```diff
@@ -56,6 +56,10 @@ public function findAll(): array
     {
         $request = "SELECT SQL_CALC_FOUND_ROWS contact_id, contact_name FROM contact WHERE contact_register = 0";
 
+        // Search
+        $searchRequest = $this->sqlRequestTranslator->translateSearchParameterToSql();
```

Comment:
I think you need to set concordance to set which field is searchablr

Reply:
Indeed, i'll synchronize with front-end team to define which field we want to search